src/applications/pensions/config/chapters/01-applicant-information/index.js

```diff
@@ -0,0 +1,12 @@
+import mailingAddress from './mailingAddress';
+import applicantInformation from './applicantInformation';
+import contactInformation from './contactInformation';
+
+export default {
+  title: 'Applicant information',
+  pages: {
+    applicantInformation,
+    mailingAddress,
+    contactInformation,
```

Comment:
The only drawbacks I potentially see with this pattern are the ability to scan a chapter's pages from within a single file. For instance, as it is today, you can view all of the chapter titles and paths from a single file (form.js). We could preserve this by defining each page in the chapter index file, while still reducing the size of the form.js file. Otherwise, this feels like a lot of clicking to scan through chapter contents.
```suggestion
    applicantInformation: {
      path: 'applicant/information',
      title: 'Applicant information',
      uiSchema: applicantInformation.uiSchema,
      schema: applicantInformation.schema,
    },
    mailingAddress: {
      path: 'applicant/address',
      title: 'Mailing address',
      uiSchema: applicantInformation.uiSchema,
      schema: applicantInformation.schema,
    },
    contactInformation: {
      path: 'applicant/contact',
      title: 'Contact information',
      uiSchema: applicantInformation.uiSchema,
      schema: applicantInformation.schema,
    },
```

Reply:
My concern with a documentation file is that documentation tends to get out-of-sync and adds maintenance burden. I like @ToddWebDev's idea of per-chapter index files that export an object with all the paths and titles visible at a glance.